source/retryable-reads/tests/listCollectionObjects.yml

```diff
@@ -0,0 +1,61 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data: []
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "ListCollectionObjects succeeds on first attempt"
+        clientOptions:
+            retryReads: true
+        operation: &retryable_operation
+            name: "listCollectionObjects"
```

Comment:
Does this need to be quoted? Just a small inconsistency when looking at all tests together.